website/datataking.html

```diff
@@ -0,0 +1,293 @@
+
+<p><span class="boldtext">Converting bytes to kilobytes: </span>
+<br><img src="https://astrochart.github.io/website/tutorials/size8.png" class="left" alt="Data Size" style="height:100px;border-radius:5px;"></br>
+
+<span class="boldtext">Congratulations, you completed this tutorial and proved that the amount of data you received is correct.</span>
+
+  <a class="anchor" id="package"></a>
+  <h2 style="text-align:left;">Take data with installable package</h2>
+  <br>While using GNUradio can be a great way to understand how data taking works and allows you to look at your data in real time, another method to take and 
+save data is by using the installable CHART package at 
+<a href="https://github.com/astrochart/CHART"> the CHART github repository.</a> with installation instructions in the readme.md file. 
+<br>
+<p> Upon installing the package, the data taking process can be started from the daq folder, with the python script freq_and_time_scan.py. The approprtiate syntax 
+is to type python freq_and_time_scan.py --arguement=value --arguement=value --etc
+<br>
+<p> Current arguements and their defaults are as follows:
+	<br>
+
+  <p>  argument('--scan_period', default=0.5, type=float, help='Time '
+                    'between a scan and the next, in hours. Default is 0.5.')
+  <br>  argument('--total_time', default=24., type=float,
+                    help='Total time for all scans, in hours. Default is 24.')
+  <br>  argument('--freq_i', default=50., type=float, help='Starting frequency, '
+                    'in MHz. Default is 50.')
+  <br>   argument('--freq_f', default=150., type=float, help='Ending frequency, '
+                    'in MHz. Default is 150.')
+  <br>  argument('--df', default=1., type=float, help='Frequency tuning step '
+                    'size, in MHz. Default is 1.')
+  <br>  argument('--sleep_time', default=5., type=float, help='Sleep time '
+                    'between checks for next scan time, in seconds. Default is 5.')
+  <br>  argument('--veclength', default=1024, type=int, help='Vector length '
+                    '(number of channels) for spectrum estimation. Default is 1024.')
+  <br>  argument('--samp_rate', default=2., type=float, help='Sample rate '
+                    'of the radio, in MHz. Default is 2.')
+  <br>  argument('--int_length', default=100, type=int, help='Number of samples '
+                    'per integration. Default is 100.')
+  <br>  argument('--int_time', type=float, help='Integration time, in seconds.'
+                    ' Overrides the --int_length argument.')
+  <br>  argument('--nint', default=100, type=int, help='Number of integrations '
+                    'per file. Default is 100.')
+  <br>  argument('--data_dir', default=None, type=str, help='Data directory. '
+                    'Defaults to current working directory.')
```

Comment:
This format looks really weird on a website. I'd suggest a table, or something closer to the docstring:
```
optional arguments:
  -h, --help            show this help message and exit
  --scan_period SCAN_PERIOD
                        Time between a scan and the next, in hours. Default is 0.5.
  --total_time TOTAL_TIME
                        Total time for all scans, in hours. Default is 24.
  --freq_i FREQ_I       Starting frequency, in MHz. Default is 50.
  --freq_f FREQ_F       Ending frequency, in MHz. Default is 150.
  --df DF               Frequency tuning step size, in MHz. Default is 1.
  --sleep_time SLEEP_TIME
                        Sleep time between checks for next scan time, in seconds. Default is 5.
  --veclength VECLENGTH
                        Vector length (number of channels) for spectrum estimation. Default is 1024.
  --samp_rate SAMP_RATE
                        Sample rate of the radio, in MHz. Default is 2.
  --int_length INT_LENGTH
                        Number of samples per integration. Default is 100.
  --int_time INT_TIME   Integration time, in seconds. Overrides the --int_length argument.
  --nint NINT           Number of integrations per file. Default is 100.
  --data_dir DATA_DIR   Data directory. Defaults to current working directory.
```